src/main/resources/com/jenkinsci/plugins/badge/dsl/AddSummaryStep/help-icon.html

```diff
@@ -27,9 +27,9 @@
         These icons are listed in the <a href="https://plugins.jenkins.io/badge/#plugin-content-icons">plugin documentation</a> and are referenced as "<i>image-name</i>.gif".
         For example, the following are all valid references to icons included with the plugin:
```

Comment:
The plugins in the webapps directory are provided by Jenkins, as far as I know.  Optional change that is unrelated to this pull request.
```suggestion
        For example, the following are all valid references to icons that can be used with the plugin:
```